asn1/src/main/java/org/wildfly/security/asn1/util/ASN1.java

```diff
@@ -27,8 +27,11 @@
 /**
  * A class that contains ASN.1 constants and utilities.
  *
+ * @deprecated Moved to {@link org.wildfly.security.asn1}
+ *
  * @author <a href="mailto:fjuma@redhat.com">Farah Juma</a>
  */
+@Deprecated
```

Comment:
The ```util``` package should just be eliminated now that its two classes are being moved one level up